twyla/service/message.py

```diff
@@ -17,32 +17,29 @@ def __init__(self, channel, body, envelope, name):
         self.envelope = envelope
         self.name = name
 
-    async def payload(self):
+    def payload(self):
```

Comment:
I have a question here -- why is the payload method made into a blocking call?

Reply:
There was a call to `channel.drop` here which has to be async, and so the containing method also had to be async.

Reply:
Ah ok, I understand -- and you wanted the validation to be more explicit which is why it's no longer dropped when it fails validation?

Reply:
I think it's too rigid. But now I recognize validation error will cause an exception here, and the message will be neither ack'd nor rejected. I will adress this issue in another PR, as the feature that implements passing Event's to the callbacks is not yet implemented anyway.